communication/marketing-team/contributor-blogging.md

```diff
@@ -0,0 +1,35 @@
+# Upstream Marketing Blog Guidelines
+
+Blogging serves as a way to highlight and give exposure to the work that is being done by the kubernetes [community](https://github.com/kubernetes/community).
+
+## Guidelines for contributor blogs: 
+
+* Use inclusive language understandable by everyone
+* Stay on topic and stick to the facts
+* Avoid interjecting personal reactions 
+* Shy away from jargon and colloquial expressions
+* Revise your post before submission to make sure it’s unambiguous and accurate
+* Emphasize the things you want readers to remember; tell a story
+* Provide evidence and data where applicable, to back up your message
+* As the author, never talk about your employer, sell, promote, or pitch; this is about upstream community endeavours and the individuals+groups that create it.
+* Avoid ambiguity
+* Avoid including anything that lacks adequate evidence
+* Make sure to have at least one original quote and one picture
+* Much of what’s outlined already in the [website blog guidelines](https://github.com/kubernetes/community/blob/77aa44ba8a506b2d8a7110ff96f545c36db906f3/sig-docs/blog-subproject/README.md
+) owned by sig-docs
+
+## Blogger Expectations, Responsibilities, and Info
+
+1. Active bloggers should write one blog a quarter. The time commitment is typically 5-10 hours per quarter depending on the number of blog posts in the review queue.
+2. Adhere to the code of conduct, values, and principles of the project.
+3. Bloggers are expected to attend at least one upstream marketing team meeting a month or check-in to remain active. 
+4. If you receive a request to write about someone or a group of people from your employer, ask another blogger to write it. 
```

Comment:
This should probably include something like "a project from your employer". Otherwise looks really good!